app/src/androidTest/java/org/openobservatory/ooniprobe/client/OONIAPIClientTest.java

```diff
@@ -64,6 +72,51 @@ public void onError(String msg) {
     }
 
     @Test
-    public void testDeleteJsons() {
+    public void testSelectMeasurementsWithJson() {
+        Delete.table(Measurement.class);
+        addMeasurement(EXISTING_REPORT_ID, true);
+        addMeasurement(EXISTING_REPORT_ID_2, true);
+        addMeasurement(NONEXISTING_REPORT_ID, true);
+        addMeasurement(NONEXISTING_REPORT_ID, true);
+        List<Measurement> measurements = Measurement.selectMeasurementsWithJson(a);
```

Comment:
I am again confused by this `a`. It's the application, isn't it?

Reply:
yes